node/network/collator-protocol/src/collator_side/mod.rs

```diff
@@ -1023,6 +967,13 @@ pub(crate) async fn run<Context>(
 				FromOrchestra::Signal(Conclude) => return Ok(()),
 			},
 			(relay_parent, peer_id) = state.active_collation_fetches.select_next_some() => {
+				// Schedule a bit reset for this peer.
+				if let Some(authority_ids) = state.peer_ids.get(&peer_id) {
+					state.reset_bit_delays.push(ResetBitDelay::new(
+						relay_parent, authority_ids.clone(), RESET_BIT_DELAY
+					));
+				}
+
```

Comment:
This timeout is probably unnecessary because we already scheduled one on advertisement.
Though we could set this one to be shorter